Sources/Model/TransferMethod/HyperwalletTransferMethod.swift

```diff
@@ -24,115 +24,137 @@ public class HyperwalletTransferMethod: NSObject, Codable {
-    /// - bankAccountId: The bank account number, IBAN or equivalent. If you are providing an IBAN,
-    ///                  the first two letters of the IBAN must match the `transferMethodCountry`.
-    /// - bankAccountRelationship: The user's relationship with the bank account holder.
-    /// - bankAccountPurpose: The purpose of the bank account (e.g. checking, savings, etc).
-    /// - bankName: The bank name.
-    /// - bankId: The bank code or equivalent (e.g. BIC/SWIFT code).
-    /// - branchName: The branch name
-    /// - branchId: The branch code, transit number, routing number or equivalent.
-    /// - businessContactRole: The bank account holder's role in the organization.
-    /// - businessName: The name of the transfer method holder's business
-    /// - businessRegistrationCountry: The country where the transfer method holder's business is registered
-    /// - businessRegistrationId: The bank account holder's business registration number or identifier, as assigned by
-    ///                           the relevant government body.
-    /// - businessRegistrationStateProvince: The state, province or region where the bank account holder's business
-    ///                                      is registered.
-    /// - businessType: The bank account holder's business type.
-    /// - country: The bank account holder's country.
-    /// - city: The bank account holder's city.
-    /// - dateOfBirth: The bank account holder's date of birth.
-    /// - firstName: The bank account holder's first name.
-    /// - governmentId: The bank account holder's government ID number, such as a Social Security Number.
-    /// - lastName: The bank account holder's last name.
-    /// - mobileNumber: The bank account holder's cell phone number.
-    /// - phoneNumber: The bank account holder's phone number.
-    /// - postalCode: The bank account holder's postal code.
-    /// - profileType: The bank account holder's profile type.
-    /// - stateProvince: The bank account holder's state, province or region.
-    /// - cardBrand: The card brand.
-    /// - cardNumber: The 16-digit card number.
-    /// - cardType: The bank card type.
-    /// - cvv: The card security code which is embossed or printed on the card.
-    /// - dateOfExpiry: The expiration date for the card (YYYY-MM).
-    /// - email: The email address associated with the PayPal account.
     public enum TransferMethodField: String {
         /// Common transfer method fields
+        /// The datetime when the transfer method was created on.
         case createdOn
+        /// The transfer method status transition.
         case status
+        /// The transfer method identifier.
         case token
+        /// The transfer method country.
         case transferMethodCountry
+        /// The transfer method currency.
         case transferMethodCurrency
+        /// The transfer method type.
         case type
-
         /// Bank Account related fields
+        ///
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed